test/RibbonThetaVault.ts

```diff
@@ -111,37 +113,6 @@ describe("RibbonThetaVault", () => {
     },
   });
 
-  behavesLikeRibbonOptionsVault({
-    name: `Ribbon WBTC Theta Vault (Put)`,
-    tokenName: "Ribbon BTC Theta Vault Put",
-    tokenSymbol: "rWBTC-THETA-P",
```

Comment:
Removed this because we don't have a WBTC put vault, so this saves us some time when running tests